crates/optimism/consensus/src/validation/mod.rs

```diff
@@ -131,7 +131,7 @@ fn verify_receipts_optimism<R: DepositReceipt>(
         calculate_receipt_root_optimism(&receipts_with_bloom, chain_spec, timestamp);
 
     // Calculate header logs bloom.
-    let logs_bloom = receipts_with_bloom.iter().fold(Bloom::ZERO, |bloom, r| bloom | r.bloom());
+    let logs_bloom = receipts_with_bloom.iter().fold(Bloom::ZERO, |bloom, r| bloom | r.logs_bloom);
```

Comment:
cool, let's also bump all `alloy-` deps from 0.15.5 to 0.15.7

then we can use `r.bloom_ref()` here